modules/nw-multus-ipam-object.adoc

```diff
@@ -310,4 +310,72 @@ pod/whereabouts-reconciler-p4sxw 1/1 Running 0 6s
 pod/whereabouts-reconciler-rvfdv 1/1 Running 0 6s
 pod/whereabouts-reconciler-svzw9 1/1 Running 0 6s
 daemonset.apps/whereabouts-reconciler 6 6 6 6 6 kubernetes.io/os=linux 6s
+----
+
+[id="nw-multus-configuring-whereabouts-ip-reconciler-schedule_{context}"]
+== Configuring the Whereabouts IP reconciler schedule
+
+The Whereabouts IPAM CNI plugin runs the IP reconciler daily to clean up any stranded IP allocations that may result in exhausting IPs and therefore prevent new Pods from getting an IP allocated to them.
+
+The following procedure shows how to change the frequency at which the IP reconciler runs. 
+
+.Prerequisites
+* You installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
```

Comment:
Sorry for the late response, I just thought about it now while re-reviewing this just to make sure.
I believe we should add here in the prerequisites the fact that the whereabouts daemonset should be deployed, with the `whereabouts-reconciler` pods up and running.
That's because that's where 'magic' happens and without it, the presence of the ConfigMap 
would do nothing.